src/bundle/Resources/public/js/scripts/admin.location.add.customurl.js

```diff
@@ -0,0 +1,29 @@
+(function (global, doc) {
+    const modal = doc.querySelector('#ez-modal--custom-url-alias');
+    const discardBtns = [...modal.querySelectorAll('[data-dismiss="modal"]')];
+    const submitBtn = modal.querySelector('[type="submit"]');
+    const inputs = [...modal.querySelectorAll('[required="required"]')];
+    const checkboxes = [...modal.querySelectorAll('.ez-field-edit--ezboolean input')];
+    const toggleButtonState = () => inputs.forEach(input => {
+        const hasValue = input.value.trim().length !== 0;
+        const methodName = hasValue ? 'removeAttribute' : 'setAttribute';
+
+        submitBtn[methodName]('disabled', true);
```

Comment:
Isn't the disable state will depend on the last required input? I mean if there will be 5 required inputs, 4 empty and the last will be filled the button will be enabled. I think it should be like: `const disabled = inputs.every(check if an input is filled)`

Reply:
The whole `foreach` is useless. There's only one field always.

Reply:
There is only one field right now, don't know if more are not coming in the future. I should not care about it then and just fetch this single one?

Reply:
If there is extensibility point and end-developer can add more inputs you should care, otherwise it's not super important (when we will add new input, we will update JS)